pekko-connectors-sample-jdbc-to-elasticsearch/project/Dependencies.scala

```diff
@@ -1,27 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
 import sbt._
 
 object Dependencies {
   val scalaVer = "2.13.8"
-  // #deps
-  val AkkaVersion = "2.6.19"
-  val AlpakkaVersion = "4.0.0"
+
+  val pekkoVersion = "1.0.1"
+
+  val pekkoConnectorVersion = "1.0.1+12-db689989-SNAPSHOT"
```

Comment:
Use the 1.0.2 release not a snaphot

Reply:
I did not find pekko connectors 1.0.2 in the Maven repository, so I used snapshot to test this sample locally and learn pekko streams. After the official release 1.0.2, I will change the versions of pekko and pekko connectors.